dodo.py

```diff
@@ -5,13 +5,13 @@
 import re
 import time
 import os
+import sys
 from datetime import datetime
 from time import mktime
 
 
 DODO_FILE = os.path.join(os.getcwd(), 'DODO')
 VERSION = "0.99"
-username = os.path.split(os.path.expanduser('~'))[-1]
```

Comment:
The same logic to pull the system username is used, just moved down to like 322. The default system username is then used as the default value for the user in the argparser. That way when the user does not specify the -u option, the default username is still used, but a global variable of ```username``` is not necessary because ```args.user``` is set.